src/ray/core_worker/core_worker.h

```diff
@@ -321,6 +321,13 @@ class CoreWorker {
                                const std::vector<std::shared_ptr<Buffer>> &metadatas,
                                std::vector<std::shared_ptr<RayObject>> *return_objects);
 
+  /// Get a handle to an actor.
+  ///
+  /// \param[in] actor_id The actor handle to get.
+  /// \param[out] actor_handle A handle to the requested actor.
+  /// \return Status::Invalid if we don't have this actor handle.
+  Status GetActorHandle(const ActorID &actor_id, ActorHandle **actor_handle) const;
```

Comment:
I'm fine with it, but just curious, why does this need to be public for java but not python?

Reply:
I think it's because certain information is only required for java: e.g. `ActorLanguage`, `FunctionDescriptor`